src/chrome/content/rules/LinuxFoundation.xml

```diff
@@ -44,9 +36,7 @@
 	<target host="collabprojects.linuxfoundation.org" />
 	<target host="developerbugs.linuxfoundation.org" />
 	<target host="events.linuxfoundation.org" />
-	<target host="eventsstg.linuxfoundation.org" />
 	<target host="identity.linuxfoundation.org" />
-	<target host="ldn.linuxfoundation.org" />
```

Comment:
But as long as the redirects are over insecure http we should secure them anyway.